### ISSUE TITLE
Spaces in shell script which defining variable

### ISSUE THREAD

**Issue by ahwanpandey** [ssadedin/bpipe#234]
https://github.com/ssadedin/bpipe/blob/97d6f9d285b0b2a6af5d04d3b26912ae3bf90e47/bin/bpipe-slurm.sh#L205

![image](https://user-images.githubusercontent.com/8450532/38477556-9c3df8b6-3bf6-11e8-8537-3ffda1fcc3d9.png)

Should the spaces be removed here?

**Comment by ssadedin** [comment]
Indeed, it looks like they should be to me. I am surprised this isn't causing wider issues!

I will put a fix in for this. Thanks for reporting it.
